package.json

```diff
@@ -40,6 +40,7 @@
         "http-browserify": "1.7.0",
         "https-browserify": "1.0.0",
         "istanbul": "git://github.com/the-t-in-rtf/istanbul#828eb930add7bd663ca20649a917f3f3829e48c5",
+        "nock": "9.0.14",
```

Comment:
As we discussed, we may need to do a Technology Evaluation for this new dependency.

Reply:
@simonbates, after [today's discussion](https://pad.gpii.net/p/server-mock-library-evaluation-krn4nue) with @amb26 and @the-t-in-rtf :

1. [A technology evaluation](https://wiki.gpii.net/w/Technology_Evaluation_-_Mocking_library) is written up;
2. Use nock in an infusion way, which means in this pull request, the nock config is cleaned up after each test run.

This pull request is ready for more reviews.